src/client/components/SuggestedVideos/Suggested-Videos-View/index.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+const SuggestedVidesView = (props) => (
+  <div className="suggested-video-container">
+    <ul className="suggested-video-ul">
+      <h1>Up next</h1>
+      <li className="suggested-video-list">
+        <div className="suggested-video">
+          <a></a>
+          <span>1:00</span>
+        </div>
+        <p className="title">video test</p>
+        <span className="video-message">
+          <span>author-test</span>
+          <span>100 views</span>
```

Comment:
in this case the user wont be navigated to the video if he/she clicks on the span tag